### PR TITLE
Add masakari rocks

### PR DESCRIPTION
**Pull request by MylesJP**
Add rocks for masakari-api, engine, and consolidated.

### CODE REVIEW

**Review by gboutry — APPROVED** [review]

File: rocks/masakari-consolidated/rockcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+package-repositories:
+  - type: apt
+    cloud: caracal
+    priority: always
+
+services:
+  wsgi-masakari-consolidated:
+    override: replace
+    command: apachectl -D FOREGROUND
```
Comment (gboutry) [inline rocks/masakari-consolidated/rockcraft.yaml:21]: Actually, can you remove this service definition in the consolidated rock? thank you

**Review by gboutry — CHANGES_REQUESTED** [review]

File: rocks/masakari-consolidated/rockcraft.yaml (hunk 1 above)
Comment (MylesJP) [reply rocks/masakari-consolidated/rockcraft.yaml:21]: For sure. Keep it in engine and API?

Comment (gboutry) [reply rocks/masakari-consolidated/rockcraft.yaml:21]: Can you rename the `wsgi-masakari-engine` into `masakari-engine`

Comment (MylesJP) [reply rocks/masakari-consolidated/rockcraft.yaml:21]: Do we want to leave "wsgi-masakari-api" named as-is?

Comment (gboutry) [reply rocks/masakari-consolidated/rockcraft.yaml:21]: Yeah, this one is ok

**Review by gboutry — APPROVED** [review]

**Comment by gboutry** [comment]
LGTM, thank you!

**Pull request merged**